msteams-platform/bots/streaming-ux.md

```diff
@@ -574,6 +587,7 @@ The following are the success and error codes:
 | `403`| `ContentStreamNotAllowed` | `Content stream finished due to exceeded streaming time.`| The bot failed to complete the streaming process within the strict time limit of two minutes. |
 | `403`| `ContentStreamNotAllowed` | `Message size too large`| The bot sent a message that exceeds the current [message size](~/bots/how-to/format-your-bot-messages.md) restriction. |
 | '403' | `ContentStreamNotAllowed` | Content stream was canceled by user. | The streaming was stopped by the user. |
+| '403' | `ContentStreamNotAllowed` | Request streamed content should contain the previously streamed content. | The incoming content for the stream message does not contain what has been already streamed. |
```

Comment:
Formatting of the error code table isn't consistent. Could you update the table so that all the error code have the correct and consistent format?
![image](https://github.com/user-attachments/assets/5761fa5d-f474-4e83-a95c-84e1ec9578c7)